keychain-environment.el

```diff
@@ -49,6 +49,29 @@
 
 ;;; Code:
 
+(defvar keychain-should-inherit
+  nil
+  "Should the keychain invocation should inherit.
+Can be either nil, local, any or t. See keychain manual.")
+
+(defvar keychain-be-quick
+  t
+  "Should keychain use the -Q option. I think there is a bug in
+keychain somewhere that means ssh-agent ends up being spawned anyway if
+  -Q is no specified")
+
+(defun keychain-command (type)
+  "Return keychain invocation for `TYPE'."
```

Comment:
TYPE shouldn't be in quotes. Also if I didn't already know what that means this description wouldn't help me much.